doc(release): prepare release notes for OSS 23.10 (#2825)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx

## Centreon Web

### 23.10.1

Release date: `November 17, 2023`

<details open>
  <summary>Enhancements</summary>

- [API] Added new endpoint to list all metrics of a given service.
- [Graph] Thickened curves.
- [Widgets] simplified resource selection.
- [UI] Updated dashboard global refresh icon.
- [UX] Made the widget edition more accessible.
- [UX] Displayed warning message when dashboard modifications have not been saved yet.
- Added metrics by service listing endpoint to service resource type.

</details>

<details>
  <summary>Bug fixes</summary>

> - [API] Fixed issue when using special characters in metric name.

Reviewer: ```suggestion
- [API] Fixed issue occurring when using special characters in metric names.
```

- [API] Fixed an issue with password clearly displayed in Host & Service API response.
- [Authentication] Fixed match with attribute used to identify users.
- [Authentication] Enabled GUI access option for imported users authenticated through SSO.

> - [Core] Fixed access to pages when Symfony debug mode is enabled.

Reviewer: ```suggestion
- [Core] Fixed access to pages when Symfony debug mode was enabled.
```

- [Event Logs] Fixed pagination.

> - [Packaging] Updated centreontrapd mode on debian central.

Reviewer: ```suggestion
- [Packaging] Fixed an issue where centreontrapd was configured for a poller rather than for a central on Debian.
```

> - [Widget] Fixed issue when loading widget.

Reviewer: ```suggestion
- [Widget] Fixed an issue that prevented a widget from loading correctly.
```

> - Fixed crash in partitioning script when database server does not support partitioning.

Reviewer: ```suggestion
- Fixed crash caused by partitioning script when database server did not support partitioning.
```

> - Fixed widgets get cloned when creating a new dashboard while editing a dashboard.

Reviewer: ```suggestion
- Fixed an issue where widgets were cloned in a new dashboard that was created while another dashboard was being edited at the same time.
```

</details>

<details open>
  <summary>Security fixes</summary>

> - Fixed potential XSS vulnerability in listing of discovered hosts.

Reviewer: ```suggestion
- Fixed potential XSS vulnerability in the list of discovered hosts.
```
